apps/myaccount/src/components/profile/profile.tsx

```diff
@@ -799,72 +828,72 @@ export const Profile: FunctionComponent<ProfileProps> = (props: ProfileProps): R
                                                 <Placeholder><Placeholder.Line /></Placeholder>
                                             )
                                             : profileInfo.get(schema.name)
-                                            ? (
-                                                schema.name === "emails" && isEmailPending
-                                                    ? (
-                                                        <>
-                                                            <p>
-                                                                {
-                                                                    profileInfo.get(schema.name)
-                                                                }
-                                                                <Popup
-                                                                    size="tiny"
-                                                                    trigger={
-                                                                        <Icon
-                                                                            name="info circle"
-                                                                            color="yellow"
-                                                                        />
+                                                ? (
+                                                    schema.name === "emails" && isEmailPending
```

Comment:
Lets get "emails" from `ProfileConstants.SCIM2_SCHEMA_DICTIONARY.get("EMAILS")` in @wso2is/core`

Reply:
Fixed by 8aee86071b94262b84aadd558d1d5376375d4927